project/Dependencies.scala

```diff
@@ -17,7 +17,7 @@ object Dependencies {
   val munitCatsEffectV = "1.0.7"
   val pact4sV = "0.10.0"
 
-  private val workbenchLibsHash = "59e30fb"
+  private val workbenchLibsHash = "2147824"
```

Comment:
Should this be the most recent hash? `9e8bb8d` (taken from [here](https://github.com/broadinstitute/workbench-libs/commit/19ae865711bf4f11787dee243676f01579338a1a))

Reply:
Yep! I don't think that functionally matters but might as well. now that it's published :)

Reply:
oh yeah true, won't really make a difference, feel free to ignore